README.md

```diff
@@ -42,3 +42,7 @@ python run_benchmark.py --help
 ```
 
 If you want to run a battery of different settings, some shell scripts are provided to achieve that. Use `run_experiments.sh` if you have a server GPU like an H100. Use `run_exps_rtx_4090.sh` if you have a consumer GPU with 24 GB of memory, like an RTX 4090. The benchmark data and sample images are stored by default in the `results/` directory.
+
+## Standalone script
+
+The `inference_lora.py` script implements the optimizations in sequence and it is geared towards H100. Refer to it for a simpler reference than `run_benchmark.py`.
```

Comment:
```suggestion
The `inference_lora.py` script implements the optimizations in sequence and it is geared towards an H100. Refer to it for a simpler reference than `run_benchmark.py`.
```

I think we could say that users should refer to it if they're only interested in applying the optimizations for image generation without the benchmarking code, WDYT?